### PR TITLE
Adds ReadEvents workload for mlmd_bench.

### PR DESCRIPTION
**Pull request by BrianSong**
Supports benchmarking the performance of GetEventsByArtifactIDs() and GetEventsByExecutionIDs() APIs.

### CODE REVIEW

File: ml_metadata/tools/mlmd_bench/proto/mlmd_bench.proto [hunk 1]
```diff
@@ -197,6 +197,20 @@ message ReadNodesViaContextEdgesConfig {
   optional Specification specification = 1;
 }
 
+// Reads Events by Artifact / Execution IDs.
+message ReadEventsConfig {
+  enum Specification {
+    UNKNOWN = 0;
+    EVENTS_BY_ARTIFACT_IDS = 1;
```
Comment (hughmiao) [inline ml_metadata/tools/mlmd_bench/proto/mlmd_bench.proto]: EVENTS_BY_ARTIFACT_ID

File: ml_metadata/tools/mlmd_bench/proto/mlmd_bench.proto [hunk 2]
```diff
@@ -197,6 +197,20 @@ message ReadNodesViaContextEdgesConfig {
...
+    EVENTS_BY_EXECUTION_IDS = 2;
```
Comment (hughmiao) [inline ml_metadata/tools/mlmd_bench/proto/mlmd_bench.proto]: ditto

**Review by hughmiao — APPROVED** [review]
minor comments, please fix before submit.

**Comment by BrianSong** [comment]
Will merge to the master branch after internal CLs pass.

**Pull request closed without merging**